.github/workflows/test.yml

```diff
@@ -5,32 +5,52 @@ on:
   push:
 
 jobs:
-  check:
-    name: Check
+  stable:
+    name: Test + Clippy (stable)
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: ATiltedTree/setup-rust@v1
-        with:
-          rust-version: stable
-      - run: cargo check
+      - run: rustup toolchain install stable --no-self-update
+      - run: rustup default stable
+      - run: rustup component add clippy
+      - run: RUSTFLAGS=-Dwarnings cargo test
+      - run: cargo clippy -- -Dwarnings
+      - run: RUSTFLAGS=-Dwarnings cargo build --manifest-path fuzz/Cargo.toml --all
 
-  test:
-    name: Test
+  nightly:
```

Comment:
Do we really want to run CI on nightly Rust? That seems like it would have frequent errors caused by Rust itself. Might want to use Rust's beta channel instead.